po/es.po

```diff
@@ -125,205 +129,206 @@ msgstr "Procesador"
 
 #: hardinfo/dmi_util.c:50
 msgid "System Power Controls"
-msgstr ""
+msgstr "Controles de energía del sistema"
 
 #: hardinfo/dmi_util.c:51
 msgid "Voltage Probe"
-msgstr ""
+msgstr "Prueba de voltaje"
 
 #: hardinfo/dmi_util.c:52
 msgid "Cooling Device"
-msgstr ""
+msgstr "Dispositivo de refrigeración"
 
 #: hardinfo/dmi_util.c:53
 msgid "Temperature Probe"
-msgstr ""
+msgstr "Prueba de temperatura"
 
 #: hardinfo/dmi_util.c:54
 msgid "Electrical Current Probe"
-msgstr ""
+msgstr "Prueba de corriente eléctrica"
 
 #: hardinfo/dmi_util.c:55
 msgid "Out-of-band Remote Access"
-msgstr ""
+msgstr "Acceso remoto fuera de banda"
 
 #: hardinfo/dmi_util.c:56
 msgid "Boot Integrity Services"
-msgstr ""
+msgstr "Servicios de integridad del arranque"
 
 #: hardinfo/dmi_util.c:57
 msgid "System Boot"
-msgstr ""
+msgstr "Sistema de arranque"
 
 #: hardinfo/dmi_util.c:58
 msgid "64-bit Memory Error"
-msgstr ""
+msgstr "Error de memoria de 64 bits"
 
 #: hardinfo/dmi_util.c:59
 msgid "Management Device"
-msgstr ""
+msgstr "Dispositivo de gestión"
 
 #: hardinfo/dmi_util.c:60
 msgid "Management Device Component"
-msgstr ""
+msgstr "Componente de dispositivo de gestión"
 
 #: hardinfo/dmi_util.c:61
 msgid "Management Device Threshold Data"
 msgstr ""
 
 #: hardinfo/dmi_util.c:62
 msgid "Memory Channel"
-msgstr ""
+msgstr "Canal de memoria"
 
 #: hardinfo/dmi_util.c:63
 msgid "IPMI Device"
-msgstr ""
+msgstr "Dispositivo IPMI"
 
 #: hardinfo/dmi_util.c:64
 msgid "Power Supply"
-msgstr ""
+msgstr "Fuente de alimentación"
 
 #: hardinfo/dmi_util.c:65
 msgid "Additional Information"
-msgstr ""
+msgstr "Información adicional"
 
 #: hardinfo/dmi_util.c:66
 msgid "Onboard Device"
-msgstr ""
+msgstr "Dispositivo embebido"
 
 #: hardinfo/dmi_util.c:226
 msgid "Invalid chassis type (0)"
-msgstr "Clasificación maquina invalida (0)"
+msgstr "Tipo de chasis no válido (0)"
 
 #: hardinfo/dmi_util.c:227 hardinfo/dmi_util.c:228
 msgid "Unknown chassis type"
-msgstr "Clasificación mauqina desconocida"
+msgstr "Tipo de chasis desconocido"
 
 #: hardinfo/dmi_util.c:229
 msgid "Desktop"
-msgstr "PC Escritorio"
+msgstr "Ordenador de escritorio"
 
 #: hardinfo/dmi_util.c:230
 msgid "Low-profile Desktop"
-msgstr "PC economico"
+msgstr "Ordenador de bajo rendimiento"
 
 #: hardinfo/dmi_util.c:231
 msgid "Pizza Box"
-msgstr "PC caja pizza"
+msgstr "Caja de pizza"
 
 #: hardinfo/dmi_util.c:232
 msgid "Mini Tower"
-msgstr "Torre mediana"
+msgstr "Minitorre"
 
 #: hardinfo/dmi_util.c:233
 msgid "Tower"
-msgstr "Torreta"
+msgstr "Torre"
 
 #: hardinfo/dmi_util.c:234
 msgid "Portable"
-msgstr "Portable"
+msgstr "Portátil"
 
 #: hardinfo/dmi_util.c:235 modules/computer.c:391 modules/computer.c:400
 #: modules/computer.c:422
 msgid "Laptop"
-msgstr "Portatil"
+msgstr "Portátil"
 
 #: hardinfo/dmi_util.c:236
 msgid "Notebook"
-msgstr "Mini portatil"
+msgstr "Portátil"
 
 #: hardinfo/dmi_util.c:237
 msgid "Handheld"
-msgstr "Mini bolsillo"
+msgstr "Ordenador de bolsillo"
```

Comment:
Those are some arm based where postmarked os are deployed! so  may be confused we need to investigate it more!

Reply:
I haven't heard "Handheld" for a long time. They were named "handheld" or "ordenador de bolsillo" IIRC. It was also a kind of portable consoles AFAIK, but it seems that now it has a different meaning by reading your description.

Reply:
so then .. as Linux can be deploy in a playstation as of a NGP etc etc devices.. it's better "de bolsillo" o "Portable"

Reply:
Ok.